pkg/common-controller/snapshot_controller.go

```diff
@@ -29,6 +29,7 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/client-go/kubernetes/scheme"
 	ref "k8s.io/client-go/tools/reference"
+	corev1 "k8s.io/component-helpers/scheduling/corev1"
```

Comment:
`corev1` is usually used for the v1 core Kubernetes API. This package here gets imported as `corev1helpers` or `v1helpers` in k/k.

Reply:
Changed to corev1helpers